demo/server/src/modules/index.ts

```diff
@@ -70,7 +71,8 @@ function wfmApiSetup(app: express.Express, connectionPromise: Promise<any>) {
   // Mount api
   const api = new WfmRestApi();
   const role = config.security.adminRole;
-  app.use('/api', securityMiddleware.protect(role), api.createWFMRouter());
+  app.use('/api', securityMiddleware.protect(role));
```

Comment:
ping @JameelB - looks like bug in security middleware prevent from getting proper results. 
This looks like workaround for the moment but we will need to get this tested.